clang/test/SemaSYCL/constexpr-function-pointer.cpp

```diff
@@ -1,10 +1,13 @@
 // RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -sycl-std=2020 -std=c++17 %s
+// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -verify=host -sycl-std=2020 -std=c++17 %s
 
 // This test checks that the compiler doesn't emit an error when indirect call
 // was made through a function pointer that is constant expression, and makes
 // sure that the error is emitted when a function pointer is not a constant
 // expression.
 
+// host-no-diagnostics
```

Comment:
Sorry, what I meant was calling through a function pointer but from the part of code that is considered host code (for example a function not marked with any attributes) while still invoking device compilation.

Reply:
oh sorry, I changed the test for the test to have some called in host context